slsDetectorServers/jungfrauDetectorServer/slsDetectorFunctionList.c

```diff
@@ -2002,6 +2003,25 @@ void alignDeserializer() {
           bus_r(ADC_DSRLZR_3_REG) & (~(ADC_DSRLZR_3_RFRSH_ALGNMNT_MSK)));
 }
 
+int getFlipRows() {
+    return ((bus_r(CONFIG_REG) & CONFIG_BOTTOM_INVERT_STREAM_MSK) >>
+            CONFIG_BOTTOM_INVERT_STREAM_OFST);
+}
+
+void setFlipRows(int arg) {
+    if (arg >= 0) {
```

Comment:
What happens if the arg is smaller than 0?

Could actually rewrite the current as 

```
if (arg){
}else{
}

Reply:
thats the point. If it < 0, it will not set flip rows.. so in the client interface, when it validates, it will fail.